packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -0,0 +1,511 @@
+        isResizable: true,
+        isMultiline: true,
+        onRender: this.createRenderCellInterface('description')
+      }
+    ];
+
+    this._methodColumns = [
+      {
+        key: 'name',
+        name: 'Name',
+        fieldName: 'name',
+        minWidth: 150,
+        maxWidth: 250,
+        isCollapsible: false,
+        isRowHeader: true,
+        isResizable: true,
+        onRender: this.createRenderCellInterface('name')
+      },
+      {
+        key: 'signature',
+        name: 'Signature',
+        fieldName: 'signature',
+        minWidth: 200,
+        maxWidth: 300,
+        isCollapsible: false,
+        isResizable: true,
+        isMultiline: true,
+        onRender: this.createRenderCellSignature('typeTokens')
+      },
+      {
+        key: 'description',
+        name: 'Description',
+        fieldName: 'description',
+        minWidth: 300,
+        maxWidth: 400,
+        isCollapsible: false,
+        isResizable: true,
+        isMultiline: true,
+        onRender: this.createRenderCellInterface('description')
+      }
+    ];
+
+    this._enumColumns = [
+      {
+        key: 'name',
+        name: 'Name',
+        fieldName: 'name',
+        minWidth: 150,
+        maxWidth: 250,
+        isCollapsible: false,
+        isRowHeader: true,
+        isResizable: true,
+        onRender: this.createRenderCellEnum('name')
+      },
+      {
+        key: 'value',
+        name: 'Value',
+        fieldName: 'value',
+        minWidth: 100,
+        maxWidth: 200,
+        isCollapsible: false,
+        isResizable: true,
+        onRender: this.createRenderCellEnum('value')
+      },
+      {
+        key: 'description',
+        name: 'Description',
+        fieldName: 'description',
+        minWidth: 300,
+        maxWidth: 400,
+        isCollapsible: false,
+        isResizable: true,
+        onRender: this.createRenderCellEnum('description')
+      }
+    ];
+  }
+
+  public renderCell = (text: string, deprecated: boolean = false) => {
+    // When the text is passed to this function, it has had newline characters removed,
+    // so this regex will match backtick sequences that span multiple lines.
+    const regex = new RegExp('`[^`]*`', 'g');
+    let regexResult: RegExpExecArray | null;
+    const codeBlocks: { index: number; text: string }[] = [];
+    while ((regexResult = regex.exec(text)) !== null) {
+      codeBlocks.push({
+        index: regexResult.index,
+        text: regexResult[0]
+      });
+    }
+
+    if (codeBlocks.length === 0) {
+      return (
+        <>
+          {deprecated ? (
+            <>
+              <Text block variant="small">
+                Warning: this API is now obsolete.
+              </Text>
+              <br />
+            </>
+          ) : (
+            undefined
+          )}
+          <Text block variant="small">
+            {text}
+          </Text>
+        </>
+      );
+    }
+
+    const eltChildren = this._extractCodeBlocks(text, codeBlocks);
```

Comment:
You might consider renaming this `eltChildren` variable for future maintainers. I wasn't sure what it meant. Or provide a descriptive comment, perhaps its just me.

Reply:
Good call!